examples/admin-partitions/terraform/aws_ecs.tf

```diff
@@ -3,8 +3,8 @@ provider "aws" {
 }
 
 locals {
-  rand_suffix = random_string.rand_suffix.result
-  ecs_name    = "consul-ecs-${random_string.rand_suffix.result}"
+  rand_suffix = lower(random_string.rand_suffix.result)
```

Comment:
Making these lower-case isn't strictly required, but this should be fine and it reduces the diff with main.